src/jscpd.coffee

```diff
@@ -32,26 +33,21 @@ class jscpd
       debug: off
       files: null
       exclude: null
-    , options
```

Comment:
Here I changed to config then options so that the config file can provide defaults that can be overridden by command line options.